test/board_test.py

```diff
@@ -0,0 +1,37 @@
+import unittest
+
+from src.board import Board
+
+class BoardTest(unittest.TestCase):
+    def test_empty_board_to_s(self):
+       board = Board()
+       expected = [ 
+            #"⊱ –––––– {⋆⌘⋆} –––––– ⊰",
+            "",
+            "         |   |   ",
+            "      ---+---+---",
+            "         |   |   ",
+            "      ---+---+---",
+            "         |   |   ",
+            "",
+            #"⊱ –––––– {⋆⌘⋆} –––––– ⊰"
+       ]
+
+       self.assertEqual(expected, board.stringify_spaces())
+
+    def test_board_with_pieces(self):
+        board = Board()
+        board.spaces = ["X", " ", " ", "O", " ", "X", " ", " ", "O"]
+        expected = [
+            #"⊱ –––––– {⋆⌘⋆} –––––– ⊰",
+            "",
+            "       X |   |   ",
+            "      ---+---+---",
+            "       O |   | X ",
+            "      ---+---+---",
+            "         |   | O ",
+            "",
+            #"⊱ –––––– {⋆⌘⋆} –––––– ⊰"
+        ]
+
+        self.assertEqual(expected, board.stringify_spaces())
```

Comment:
I didn't realize we were going to be marking the board already! Isn't that part of a future story?

Reply:
Maybe I jumped ahead of myself a bit here, interpolating the spaces into the board strings seemed like a logical thing to do rather than just leaving it as spaces to start. The board's starting state is empty strings and we have no way to change that at this point in time, but the board has the capability of displaying tokens if the spaces list had them inside of it, that's why I built out this test. 

Again, I may have jumped ahead of myself a bit here...